source/code/shared/tools/scx_ssl_config/scx_ssl_config.cpp

```diff
@@ -461,6 +462,17 @@ static int DoGenerate(const wstring & targetPath, int startDays, int endDays,
         if (0 != rc) {
             throw SCXCoreLib::SCXErrnoFileException(L"chmod", keyPath.Get(), errno, SCXSRCLOCATION);
         }
+
+        struct passwd *pwd=NULL;
+        if ((pwd = getpwnam("omi")) != NULL) {
```

Comment:
Weird. Check this [page](https://linux.die.net/man/3/getpwnam). Note that I have limited resources while out of town, but this appears to be a valid manpage, and it states that errno should be set to zero before calling `getpwnam` if you want to check for error afterwards.

Please check a "real" manpage on one of our systems and, if it agrees, please set errno to zero before the call to `getpwnam`. This is odd, as I've never noted that before. But if that's what the manpage says, then that's what the manpage says.

Reply:
Thanks for pointing this out. In real man page also it has mentioned in same lines. I will add errno=0 prior to call getpwnam.